frontend/public/components/resource-quota.jsx

```diff
@@ -85,16 +85,15 @@ const UsageIcon = ({capacity}) => {
 const getDecimalUsage = (used, max, resourceType) => {
   let capacity = 0;
   if (max && used) {
-    const unitType = units.getUnitType(max) || units.getUnitType(used);
-    // max cpu values do not have units, convert number and add unit for capacity calculation
-    // Ex. converts '1' to '1000m'
-    let maxStr = max;
-    let maxNum = _.toNumber(max);
-    if (_.includes(resourceType, '.cpu') && unitType && !_.isNaN(maxNum)) { // skip if max cpu already has a unit
-      maxStr = `${maxNum * 1000}m`;
+    let usedNum = 0, maxNum = 0;
+    if (_.includes(resourceType, '.cpu')) {
```

Comment:
I'm a little uneasy about the `getUnitType` detection. I'm not sure we can reliably detect.

I suggest creating sets of known CPU and memory/size quota resources and calling the right dehumanize method for those.

Reply:
You should be able to use #852 to get the base values now without needing to check unit types.